plugin.json

```diff
@@ -6,7 +6,7 @@
     "support_url": "https://github.com/mattermost/mattermost-plugin-github/issues",
     "icon_path": "assets/icon.svg",
     "version": "1.0.0",
-    "min_server_version": "5.12.0",
+    "min_server_version": "5.25.0",
```

Comment:
Why so high? `PluginHTTP` only needs 5.18.

Reply:
Because the bug fix in https://github.com/mattermost/mattermost-server/pull/14905 is needed for the plugin to work.